config/config.go

```diff
@@ -78,3 +96,38 @@ func Translate(input []byte, options common.TranslateOptions) ([]byte, report.Re
 
 	return translator(input, options)
 }
+
+// MergeBytes translates and merges a slice of raw configs to a single raw
+// Ignition config. The variant and version of the first raw config
+// ("fragment") determines the merger version to use.
+func MergeBytes(fragments [][]byte, options common.TranslateOptions) ([]byte, error) {
```

Comment:
Is report merging a thing?

Reply:
Yes, there is `Report.Merge()` for that.